spring-integration-jdbc/src/test/java/org/springframework/integration/jdbc/config/pollingWithSelectParameterSourceJdbcInboundChannelAdapterTest.xml

```diff
@@ -15,17 +15,22 @@
 			update="delete from item" />
 		
 	<beans:import  resource="jdbcInboundChannelAdapterCommonConfig.xml" />
-	
+
 	<beans:bean id="parameterSource" factory-bean="parameterSourceFactory" factory-method="createParameterSourceNoCache">
-		<beans:constructor-arg value="" />
+		<beans:constructor-arg value="#{ T(java.sql.Types).INTEGER}" />
```

Comment:
What is the point of this change, please?

Reply:
By mistake, will fix in the next commit.